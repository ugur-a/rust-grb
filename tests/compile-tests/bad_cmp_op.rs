use grb::prelude::*;
mod utils;

fn main() -> grb::Result<()> {
    create_model!(_g, m, x, y, z);
    c!(x + y != 1 - z);
    c!(x + y < 1 - z);
    c!(x + y > 1 - z);
    Ok(())
}
