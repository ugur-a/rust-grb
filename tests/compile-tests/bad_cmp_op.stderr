error: expected >=, <= or ==
 --> tests/compile-tests/bad_cmp_op.rs:6:14
  |
6 |     c!(x + y != 1 - z);
  |              ^^

error: expected >=, <= or ==
 --> tests/compile-tests/bad_cmp_op.rs:7:14
  |
7 |     c!(x + y < 1 - z);
  |              ^

error: expected >=, <= or ==
 --> tests/compile-tests/bad_cmp_op.rs:8:14
  |
8 |     c!(x + y > 1 - z);
  |              ^
